### PR TITLE
Mark vocal as EOL

### PR DESCRIPTION
**Pull request by ghost**
> This repository has been archived by the owner on Sep 3, 2024. It is now read-only.

Source: https://github.com/VocalPodcastProject/vocal

### CODE REVIEW

**Comment by nathandyer** [comment]
Hi @yakushabb! Thanks for this PR. Vocal is not EOL - it has just moved to Codeberg: https://codeberg.org/nathandyer/Vocal 

We're planning a new release, which will have a new App ID that will supercede the current one, at which point we'll update the manifest with the appropriate `end-of-life-rebase` information.

**Comment by nathandyer** [comment]
Hello @yakushabb, I've reopened this PR because I have made the decision to go ahead and end the project, rather than push forward with a new release as I originally planned. Sorry for the back-and-forth on this, as you can imagine it wasn't an easy decision to make.

**Comment by ghost** [comment]
@nathandyer and @bbhtt ready for review and merge.

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "end-of-life": "This application is no longer maintained."
```
Comment (bbhtt) [inline flathub.json:2]: eol-rebase, needs to wait for the new submission.

> We're planning a new release, which will have a new App ID that will supercede the current one, at which point we'll update the manifest with the appropriate end-of-life-rebase information.

Comment (ghost) [reply flathub.json:2]: He decided "Vocal is End-of-Life"

https://nathandyer.me/2024/11/26/end-of-vocal.html

**Comment by nathandyer** [comment]
Thank you @yakushabb!

**Pull request merged**